lib/testing/src/lib/content-services/pages/like.page.ts

```diff
@@ -0,0 +1,97 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { browser, by, element } from 'protractor';
+import { BrowserVisibility } from '../../core/utils/browser-visibility';
+
+export class LikePage {
+
+    nodeIdField = element(by.css(`input[id="nodeId"]`));
```

Comment:
This locator is part of the demo-shell page. Please create a new page in demo-shell folder and move this locator there.